app/partials/content/Accordion.html

```diff
@@ -6,9 +6,10 @@
                 <span class='ru-level-{{pageSubject}}-{{c.level || "0"}}'></span>
                 <img src='/assets/icon-expand-arrow.png'>
             </div>
-            <span class="accordion-section-title-suffix" ng-bind="titleSuffixes[$index]" math-jax></span>
+            
+            <span class="accordion-section-title-suffix" ng-bind="titleSuffixes[$index]" math-jax>{{titleSuffixes[$index]}}</span>
```

Comment:
What's this change for, and why?

Reply:
No idea, I forgot to ask about this. I've reverted it as it is unrelated to the branch.